designer/server/src/main/scala/pl/touk/nussknacker/ui/util/AkkaToTapirStreamExtension.scala

```diff
@@ -0,0 +1,53 @@
+package pl.touk.nussknacker.ui.util
+
+import akka.stream.Materializer
+import akka.stream.scaladsl.{Source, StreamConverters}
+import akka.util.ByteString
+import sttp.capabilities.Streams
+import sttp.capabilities.akka.AkkaStreams
+import sttp.tapir.CodecFormat.OctetStream
+import sttp.tapir.EndpointIO.StreamBodyWrapper
+import sttp.tapir.{Codec, CodecFormat, DecodeResult, EndpointIO, Schema, StreamBodyIO}
+
+import java.io.InputStream
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.FiniteDuration
+
+class AkkaToTapirStreamExtension(implicit val mat: Materializer) {
```

Comment:
We are planning to get rid of akka backend. Can we make this solution ready for this step or migration of this will be a part of switching to another backend?